src/loxone-handler-base.ts

```diff
@@ -219,6 +219,36 @@ export abstract class LoxoneHandlerBase {
         await this.updateStateObjectAsync(uuid + '.' + this.normalizeName(name), common, uuid);
     }
 
+    protected async createTextInputStateObjectAsync(
```

Comment:
Methods that are only used in exactly one control, should not be in the base class. Please move (or inline) this method to `TextInput`.